packages/components/pie-button/src/index.ts

```diff
@@ -43,6 +58,95 @@ export class PieButton extends LitElement implements ButtonProps {
     @property({ type: Boolean })
     public isFullWidth = false;
 
+    @property({ type: String })
+    public name?: string;
+
+    @property({ type: String })
+    public value?: string;
+
+    @property()
+    public formaction: ButtonProps['formaction'];
+
+    @property()
+    public formenctype: ButtonProps['formenctype'];
+
+    @property()
+    public formmethod: ButtonProps['formmethod'];
+
+    @property({ type: Boolean })
+    public formnovalidate: ButtonProps['formnovalidate'];
+
+    @property()
+    public formtarget: ButtonProps['formtarget'];
+
+    /**
+     * This method creates an invisible button of the same type as pie-button. It is then clicked, and immediately removed from the DOM.
+     * This is done so that we trigger native form actions, such as submit and reset in the browser. The performance impact of adding and removing a single button to the DOM
+     * should be neglible, however this should be monitored.
+     * This is the only viable way of guaranteeing native button behaviour when using a web component in place of an actual HTML button.
+     *
+     * TODO: if we need to repeat this logic elsewhere, then we should consider moving this code to a shared class or mixin.
+     */
+    private _simulateNativeButtonClick (btnType: 'submit' | 'reset') {
+        if (!this.form) return;
+
+        const btn = document.createElement('button');
+        btn.type = btnType;
+
+        // Visually hidden styles
+        btn.style.position = 'absolute';
+        btn.style.width = '1px';
+        btn.style.height = '1px';
+        btn.style.padding = '0';
+        btn.style.margin = '-1px';
+        btn.style.overflow = 'hidden';
+        btn.style.border = '0';
+        btn.style.whiteSpace = 'nowrap';
+
+        if (btnType === 'submit') {
+            if (this.name) {
+                btn.name = this.name;
+            }
+            if (this.value) {
+                btn.value = this.value;
+            }
+            if (this.formaction) {
+                btn.setAttribute('formaction', this.formaction);
+            }
+            if (this.formenctype) {
+                btn.setAttribute('formenctype', this.formenctype);
+            }
+            if (this.formmethod) {
+                btn.setAttribute('formmethod', this.formmethod);
+            }
+            if (this.formnovalidate) {
+                btn.setAttribute('formnovalidate', 'formnovalidate');
```

Comment:
Should this have been this instead?
```suggestion
                btn.setAttribute('formnovalidate', this.formnovalidate);
```